### PR TITLE
feat(interpreter): export utility macros

### PR DESCRIPTION
**Pull request by DaniPopes**
Extends https://github.com/bluealloy/revm/pull/1201

### CODE REVIEW

**Review by mattsse — APPROVED** [review]
lgtm

File: crates/interpreter/src/instructions/macros.rs [hunk 1]
```diff
@@ -29,12 +37,16 @@ macro_rules! gas {
...
+/// Records a `gas` refund.
+#[macro_export]
 macro_rules! refund {
     ($interp:expr, $gas:expr) => {
         $interp.gas.record_refund($gas)
     };
 }
 
+/// Same as [`gas!`], but with `gas` as an option.
+#[macro_export]
```
Comment (fgimenez) [inline crates/interpreter/src/instructions/macros.rs:49]: This macro calls `gas!`, if we want to consume it externally without importing also `gas!` we might need to change the call to `$crate::gas!`. Same for `as_usize_or_fail!` and `as_usize_or_fail_ret!`

**Review by rakita — APPROVED** [review]
lgtm! Pending @fgimenez comment

**Comment by DaniPopes** [comment]
Done

**Review by fgimenez — APPROVED** [review]

**Pull request merged**